storey/sources.py

```diff
@@ -875,3 +875,124 @@ def _init(self):
             else:
                 df = pandas.read_parquet(path, columns=self._columns, storage_options=self._storage_options)
             self._dfs.append(df)
+
+    class MongoDBSource(_IterableSource, WithUUID):
+        """Use mongodb collection as input source for a flow.
+
+        :parameter key_field: string. column to be used as key for events. can be list of columns
```

Comment:
The parameter types are annotated, so no need to put it in the class comment. Also the comment doesn't match the type annotation.